_posts/blog/2020-07-17-measurement-observations-on-network-performance-during-the-COVID-19-pandemic-in-Northern-Italy.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: blog
+title: "Measurement Observations on Network Performance During the COVID-19 Pandemic in Northern Italy"
+author: " Simone Basso (OONI), Chris Ritzo (M-Lab), Maria Xynou (OONI)"
+date: 2020-07-17
+breadcrumb: blog
+categories:
+  - data
+  - bigquery
+  - research
+  - covid19
+---
+
+We were recently invited to participate in a [NetGain Partnership webinar](https://www.netgainpartnership.org/){:target="_blank"} (titled “[Surging Demand and The Global Internet Infrastructure](https://www.netgainpartnership.org/events/covid19/global-infrastructure){:target="_blank"}”) to discuss the changing landscape for internet infrastructure and technology in the wake of the COVID-19 pandemic.
+ 
+As part of our preparation for this webinar, we looked at network performance measurements collected from northern Italy over the last months (i.e. when Italy was hit hard by the COVID-19 pandemic) in an attempt to understand whether and to what extent there was a correlation between increased internet use and reduced network performance. As our observations may be of public interest, we decided to share them through this blog post.<!--more-->
+
+In summary, we observe:
+ 
+* Significant increase in the number of NDT tests run per day around the beginning of Italy’s nationwide lockdown (which started on 10th March 2020);
+* Reduced NDT download and upload speed results during the nationwide lockdown in Italy (between March 2020 to May 2020);
+* Improved NDT download and upload speed results from early May 2020 onwards, when the Italian government gradually started to lift the strict lockdown measures.
+
+Further details are shared through this report.
+
+* The NDT speed test
+* Data analysis
+* Conclusion
```

Comment:
These should either be links or cancelled. To make them links something like `[The NDT speed test](#the-ndt-speed-test)` would do.